parquet-column/src/main/java/org/apache/parquet/schema/LogicalTypeAnnotation.java

```diff
@@ -147,6 +147,12 @@ protected LogicalTypeAnnotation fromString(List<String> params) {
         return float16Type();
       }
     },
+    VARIANT {
```

Comment:
This should be working because it was added in https://github.com/apache/parquet-java/pull/3072